server/conf/messages

```diff
@@ -322,7 +322,7 @@ content.confirmed=Thank you for applying to {0}.  Your application has been rece
 title.createAnAccount=Create an account or sign in
 
 # A message urging users to create an account to save their data.
```

Comment:
Suggestion:blocking I would recommend finding out a bit more about the translation process to understand what the person who translates text gets to see (this file?). In that place I would document well what the value that is passed in represents and how it fits into the context of the sentence. The goal would be that the person who does the translations fully understands what they are translating. This is important for example in languages where the word order is different because translating in sections can result in a sentence with a completely different meaning

suggestion (optional)
Big kudos if you also want to update our process a bit to make this the standard and prevent future translation errors (could be done separately, but would be quick while you are here)

I'm thinking along the lines of adding a comment such as:

This files contains the strings used in applicant UI as they are seen by translators. To make correct translations as easy as possible, please include a description that allows someone without knowledge about where this string is being used to understand it fully. For example explain where the string is shown and if strings are deployment specific (e.g. they use the civic entities name), specify what each passed through value represents in the following format...

And then updating the comments of the  few strings where the {0} notation is already used

Reply:
Confirmed with Austin that the translators do see the description. Added a sentence describing what {0} represents, and added a paragraph similar to your suggestion to the top of the file.